### PR TITLE
feat(route/copernicium): Add default

### PR DESCRIPTION
**Pull request by dzx-dzx**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/copernicium
/copernicium/环球视角
/copernicium/专题报道
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by dzx-dzx** [comment]
The site's SSL is misconfigured, which is the source of problem here.

File: lib/routes/copernicium/index.ts [hunk 1]
```diff
@@ -17,25 +17,32 @@ export const route: Route = {
...
+                _jcp: CATEGORY_TO_ARG_MAP.get(ctx.req.param().category),
+                m31pageno: 1,
+            },
+            type: 0,
+        };
+        res = await ofetch(`http://www.copernicium.tw/nr.jsp`, {
+            query: { _reqArgs: reqArgs },
+        });
+    } else {
+        res = await ofetch(`http://www.copernicium.tw/sys-nr/`, {
```
Comment (TonyRL) [inline lib/routes/copernicium/index.ts:42]:
```suggestion
        res = await ofetch('http://www.copernicium.tw/nr.jsp', {
            query: { _reqArgs: reqArgs },
        });
    } else {
        res = await ofetch('http://www.copernicium.tw/sys-nr/', {
```

File: lib/routes/copernicium/index.ts [hunk 2]
```diff
@@ -17,25 +17,32 @@ export const route: Route = {
...
-    const CATEGORY_TO_ARG_MAP = new Map([
-        ['环球视角', '4_1'],
-        ['人文叙述', '4_3'],
-        ['观点评论', '4_5'],
-        ['专题报道', '4_7'],
-    ]);
-    if (!CATEGORY_TO_ARG_MAP.get(ctx.req.param().category)) {
-        throw new Error('The requested category does not exist or is not supported.');
+    let res;
+    if (ctx.req.param('category')) {
```
Comment (TonyRL) [inline lib/routes/copernicium/index.ts:21]: Please declare a variable for `category` as it is referenced multiple times

File: lib/routes/copernicium/index.ts [hunk 3]
```diff
@@ -60,8 +68,8 @@ async function handler(ctx) {
         )
     );
     return {
-        title: `日新说 - ${ctx.req.param().category}`,
-        link: 'https://www.copernicium.tw',
+        title: `日新说 - ${ctx.req.param().category ?? '全部文章'}`,
```
Comment (TonyRL) [inline lib/routes/copernicium/index.ts:71]:
```suggestion
        title: `日新说 - ${category ?? '全部文章'}`,
```

**Pull request merged**